challenges/objects-arrays.js

```diff
@@ -46,24 +38,27 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
-/* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
-
-Once you have the new array created, sort the universities alphabetically and log the result. */
+// Request 1: Create a new array called universities that contains all the universities in the graduates array.  
+// Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities)
-
-/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
+graduates.forEach(element => {let uni_list = element.university; 
+  universities.push(uni_list);});
+console.log(universities.sort());
 
-The resulting contact information should have a space between the first name and the email information like this: 
-Name email@example.com
-
-Log the result of your new array. */
+// Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 const contactInfo = [];
+graduates.forEach(element => {let Info = `${element.first_name} ${element.email}`;
+  contactInfo.push(Info);
+});
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+let filter_uni = graduates.filter(element => {return (element.university === 'Uni');
+});
+uni.push(filter_uni)
```

Comment:
A good trick to putting the contents of one array inside another is to use the spread operator `...`:
```javascript
const uni = [...filter_uni];
```
This will "spread" the insides of the `filter_uni` array inside the `uni` array! You can also do this with objects 👍